technische-universitat-dresden-linguistik.csl

```diff
@@ -27,6 +27,11 @@
     </terms>
   </locale>
   <macro name="creator-year-long">
+    <choose>
+      <if match="none" variable="author editor">
+        <text value="o. A." suffix=" "/>
+      </if>
+    </choose>
```

Comment:
@fussballlinguist, this is fine, but it's slightly nicer to use `<substitute/>` to print "o. A.". There is also a dedicated "anonymous" term that can be used to print "o. A.".

Example of a style using `<substitute/>` as discussed above: https://github.com/citation-style-language/styles/blob/6152ccea8b7d7a472910d36524d1bf3557a83bfc/im-gesprach.csl#L43-L55
The German CSL locale file, which already uses "o. A." (with a non-breaking space) for the abbreviated form of the "anonymous" term: https://github.com/citation-style-language/locales/blob/ec6d62a9e7c049001a55086a495022520989441c/locales-de-DE.xml#L38

Reply:
And I'll just merge this as-is, but if you are willing to make this change please feel free to open a new pull request for this.